test/oauth2-service.test.ts

```diff
@@ -152,6 +152,24 @@ describe('OAuth 2 service', () => {
     });
   });
 
+  it('should expose a token endpoint that includes an aud claim on Client Credentials grants', async () => {
```

Comment:
@kadams54 Could this be turned into a data-driven test leveraging `it.each()` and passing `"aud"` and `["aud1", "aud2"]` as values?

Reply:
Yup. Good catch. I'll throw that in there.

Reply:
OK, so this turned out to be a little more complicated than I expected, mostly because SuperAgent turns arrays into `aud[0]=aud1&aud[1]=aud2` when sending form data. On the mock oauth2 server side, `reqBody` becomes `{ "aud[0]": "aud1", "aud[1]": "aud2" }` instead of what we want, `{ "aud": ["aud1", "aud2"] }`.

To resolve this, I overrode the `type('form')` on the request and changed it back to `type('json')`. I hope that's OK?

Reply:
@kadams54 try this:

```js
import qs from 'querystring';

// ...

const res = await tokenRequest(service.requestHandler)
  .send(qs.stringify({
    grant_type: 'client_credentials',
    scope: 'urn:first-scope urn:second-scope',
    aud,
  }))
  .expect(200);
```

Reply:
Yup, that works too.